docs/webiny-apps/headless-cms/features/access-tokens.md

```diff
@@ -0,0 +1,23 @@
+---
+id: access-tokens
+title: Access Tokens
+sidebar_label: Access Tokens
+---
+
+Access Tokens is another feature of Headless CMS :)
+
+You can find Access tokens in the menu shown in the image below:
+
+![Access tokens in menu](/img/webiny-apps/headless-cms/features/content-modeling/access-tokens-menu.png)
+
+Access tokens provide you access to certain environments.
+
+To gain access to certain environments you need to:
+
+1. Create an access token for a specific environment.
+
+![Access Tokens](/img/webiny-apps/headless-cms/features/content-modeling/access-token.png)
+
+2. Use this access token as a HTTP header to your GraphQL Playground in `/cms/read/{alias}` endpoints.
+
+![Access Tokens](/img/webiny-apps/headless-cms/features/content-modeling/access-token-graphql.png)
```

Comment:
This image should probably show the graphql layer showing a success message so the token itself actually works and they know what to expect.
![image](https://user-images.githubusercontent.com/9532882/83656489-9bbc4480-a58d-11ea-9bc4-8301d2be69f3.png)

Reply:
We will add other screenshots here when new updates are released :)